tests/interactions/focusable-test.js

```diff
@@ -15,8 +15,11 @@ describe('BigTest Interaction: focusable', () => {
   beforeEach(() => {
     focused = false;
 
-    document.querySelector('.test-input')
-      .addEventListener('focus', () => focused = true);
+    document.querySelector('.test-input').addEventListener('focus', () => {
+      if (document.querySelector('.test-input') === document.activeElement) {
+        focused = true;
```

Comment:
How about `e => focused = e.target === document.activeElement`? `if ([true]) true` is a little redundant.